docs/shop4cfdefinitions.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.0.1",
+    "$id": "https://smart-data-models.github.io/dataModel.SmartManufacturing/SHOP4CF/schema.json",
+    "title": " Smart Data Models - SHOP4CF",
+    "definitions": 
+    { 
+      "isDefinedBy": "Relationship. Model:'https://schema.org/URL'. References a Task Definition (which does not rather live in Context Broker)",
```

Comment:
I'm not sure for `Model:'https://schema.org/URL'`. I would remove this part. 

As this attribute is a Relationship, it should be the identifier of another object. And identifiers are URNs, not necessarily URLs.

In the SHOP4CF example, we have value `urn:ngsi-ld:TaskDefinition:company-xyz:hybrid-transportation-x`.

Reply:
Unfortunately, the tool in smart data model just gave me the `https://schema.org/URL` are you aware if there is a URN like definition?  According to my knowledge `https://schema.org/` does not have URNs or URIs

Reply:
Yes, it seems `schema.org` does not have URNs. 

For sure, URNs is what is used in FIWARE for identifiers. Thus for now, I suggest removing `Model:'https://schema.org/URL'`.

Later, we can do more research on how other smart data models solve this issue.

Reply:
Ok